c2corg_ui/static/js/style/floatbuttons.js

```diff
@@ -0,0 +1,32 @@
+goog.provide('app.floatButtonResponsive');
+
+goog.require('app');
+
+/**
+ * 
+ * @export
+ */
+app.floatButtonResponsive = function() {
+  // This code will watch the Title height and adapt the buttons position accordingly.
+  var detailTitleHeight = $('.view-details-title').height();
+  adaptFloatingButtons(detailTitleHeight);
+
+  // FIX if i copy a adaptFloatingButtons to setTimeout, closure says that parameter is undefined o.O
```

Comment:
I am not sure if I understand the problem. But if you do `setTimeout(adaptFloatingButtons, 200);` the parameter `detailTitleHeight` which the function expects is missing. You could move `var detailTitleHeight = $('.view-details-title').height();` into the function and then remove the parameter (or do a [partial apply](https://docs.angularjs.org/api/ng/function/angular.bind)). This would avoid some duplication.